apps/web-twig-demo/templates/partials/header.html.twig

```diff
@@ -1,17 +1,48 @@
-<header class='spirit-theme-light-on-brand bg-primary text-secondary py-1200 py-tablet-1400'>
-  <div class="Container">
+<Header UNSAFE_className="spirit-theme-light-on-brand">
 
-    <h1 class='typography-heading-xlarge-bold'>Spirit Design System</h1>
-    <h2 class='typography-heading-medium-bold text-secondary mb-1000'>Web Twig — Development Preview</h2>
+  <a href="http://localhost:3456" aria-label="Spirit Development Preview">
+    {% include 'assets/spiritLogoInverted.html.twig' %}
+  </a>
 
-    <div class="d-grid d-tablet-block">
-      <a href="https://github.com/lmc-eu/spirit-design-system" class="Button Button--tertiary Button--large mb-400 mb-tablet-0 mr-tablet-400">
-        GitHub Project
-      </a>
-      <a href="https://spirit.supernova-docs.io/spirit/" class="Button Button--tertiary Button--large">
-        Spirit Docs →
-      </a>
-    </div>
+  <HeaderMobileActions dialogId="off-canvas-navigation" />
 
-  </div>
-</header>
+  <HeaderDesktopActions aria-label="Main navigation">
+    <HeaderNav>
+      <HeaderNavItem>
+        <HeaderLink href="http://localhost:3456/packages/web/">Web</HeaderLink>
```

Comment:
It would be great if we could somehow get these URLs from the common package, where they are defined, I think.

Reply:
I mean at least the domain and port

Reply:
Filed as https://jira.almacareer.tech/browse/DS-1506. But I don't think it will be possible for Twig without converting the configs to YAML (for example).